packages/experiments/src/components/Chiclet/examples/Chiclet.Xsmall.Footer.Example.tsx

```diff
@@ -47,13 +41,9 @@ const sizeStyle = mergeStyles({
   letterSpacing: 'normal'
 });
 
-export class ChicletXsmallFooterExample extends React.Component<{}, IChicletXsmallFooterExampleState> {
+export class ChicletXsmallFooterExample extends React.Component<{}, {}> {
```

Comment:
```suggestion
export class ChicletXsmallFooterExample extends React.Component {
```

Though you might consider making this and similar components functional at this point.